modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_3_2/TemplateUpgradeProcess.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-package com.liferay.dynamic.data.mapping.internal.upgrade;
+package com.liferay.dynamic.data.mapping.internal.upgrade.v5_3_2;
```

Comment:
@velosofelipe64, makes more consistent, considering the commit's message, move all this renaming + deleting of Java classes to the next commit. Can you please do it?